src/lib/utils/courses.ts

```diff
@@ -7,3 +8,19 @@ export const getFirstSectionType = (sections: Section[], type: SectionType): num
 export const hasSectionType = (sections: Section[], type: SectionType): boolean => {
   return sections.some((section) => section.sectionType === type);
 };
+
+/*
+  Given an array of type SavedCourse[], it returns
+  an object of format
+  {
+    Term: SavedCourse[]
+  }
+  with all courses split by the term they are saved in
+*/
+export const groupCoursesBy = (courses: SavedCourse[]) => {
```

Comment:
```suggestion
export const groupCoursesByTerm = (courses: SavedCourse[]) => {
```
since it's always grouping by term?